Server/ConcordiaCurriculumManager/Controllers/CourseController.cs

```diff
@@ -70,12 +69,12 @@ public async Task<ActionResult> GetAllCourseSettings()
     [SwaggerResponse(StatusCodes.Status500InternalServerError, "Unexpected error")]
     [SwaggerResponse(StatusCodes.Status201Created, "Course creation dossier created successfully", typeof(Guid))]
     [SwaggerRequestExample(typeof(CourseCreationInitiationDTO), typeof(CourseCreationInitiationDTOExample))]
-    public async Task<ActionResult> InitiateCourseCreation([FromBody, Required] CourseCreationInitiationDTO initiation)
+    public async Task<ActionResult> InitiateCourseCreation([FromBody, Required] CourseCreationInitiationDTO initiation, Guid dossierId)
     {
         try
         {
             var user = await _userService.GetCurrentUser();
```

Comment:
Since the only thing we need from the user is the ID, there's a new better way to handle this I recently implemented, see the example in GetDossiersByID https://github.com/JamesPartsafas/ConcordiaCurriculumManager/blob/main/Server/ConcordiaCurriculumManager/Controllers/DossierController.cs#L45